harvester/HarvestRepository.py

```diff
@@ -22,7 +24,7 @@ def __init__(self, params):
 	def setDefaults(self, defaults):
 		""" Inherit global configs, but they do not overwrite local configs """
 		for k,v in defaults.items():
-			if isinstance(k, str) or isinstance(k, unicode):
+			if isinstance(k, str) or isinstance(k, unicode) and not isinstance(v, list):
```

Comment:
The intent is this: add all configuration definitions from the parent scope into this repo's scope, but don't overwrite any values that are already defined.

The current code is checking for strings in the KEYS, but I think I originally intended to check for strings in the VALUES.  So I propose to change the line to this:

if isinstance(v, str) or isinstance(v, unicode):

But then maybe I'm doing this backwards, and the global values should be put in place first, and then the local values are set after, which would eliminate all of this "checking to see if it is set" logic.